frontend/src/screens/channels/Channels.tsx

```diff
@@ -288,24 +199,200 @@ export default function Channels() {
                 </div>
                 <DialogFooter>
                   <LoadingButton
-                    loading={loadingAddress}
+                    loading={loadingSwap}
                     type="submit"
                     onClick={async () => {
-                      setLoadingAddress(true);
+                      setLoadingSwap(true);
                       const onchainAddress = await getNewAddress();
                       if (onchainAddress) {
                         openLink(
                           `https://boltz.exchange/?sendAsset=LN&receiveAsset=BTC&sendAmount=${swapOutAmount}&destination=${onchainAddress}&ref=alby`
                         );
                       }
-                      setLoadingAddress(false);
+                      setLoadingSwap(false);
                     }}
                   >
                     Swap out
                   </LoadingButton>
                 </DialogFooter>
               </DialogContent>
             </Dialog>
+            <Dialog open={swapInDialogOpen} onOpenChange={setSwapInDialogOpen}>
+              <DialogContent>
+                <DialogHeader>
+                  <DialogTitle>Swap in funds</DialogTitle>
+                  <DialogDescription>
+                    Swap on-chain funds into your lightning channels via a swap
+                    service, increasing your spending balance using on-chain
+                    funds from{" "}
+                    <Link to="/wallet/withdraw" className="underline">
+                      your hub
+                    </Link>{" "}
+                    or an external wallet.
+                  </DialogDescription>
+                </DialogHeader>
+                <div className="grid grid-cols-4">
+                  <Label className="pt-3">Amount (sats)</Label>
+                  <div className="col-span-3">
+                    <Input
+                      value={swapInAmount}
+                      onChange={(e) => setSwapInAmount(e.target.value)}
+                    />
+                    <p className="text-muted-foreground text-xs p-2">
+                      The amount is set to 90% of the funds held by the
+                      counterparty in the channel with the most receiving
+                      capacity.
```

Comment:
Shall we just say: The amount is set to 90% of the highest receiving capacity available in one of your lightning channels?

("held by the counterparty" sounds very advanced. I tried to use terms from the Node page)

Reply:
Maybe even better: 

The amount is set to 90% of the maximum receiving capacity available in one of your Lightning channels.

Reply:
Feel free to improve it further: 

https://github.com/getAlby/hub/pull/973